### PR TITLE
[Track2] Impl of web app manager

### PR DESCRIPTION
**Pull request by Flanker32**
- Add impl of web app manager in library
- Add empty impl of app service plan and deployment slot for compile

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/AzureAppService.java [hunk 1]
```diff
@@ -49,7 +52,7 @@ public IWebApp webapp(WebAppEntity webAppEntity) {
     }
 
     public IAppServicePlan appServicePlan(AppServicePlanEntity appServicePlanEntity) {
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/AzureAppService.java:54]: plan

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/WebAppDeploymentSlot.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import com.microsoft.azure.toolkits.appservice.service.IWebAppDeploymentSlotCreator;
+
+import java.io.File;
+
+public class WebAppDeploymentSlot implements IWebAppDeploymentSlot {
+
+    private AzureAppService azureAppService;
+    private WebAppDeploymentSlotEntity slotEntity;
+
+    private DeploymentSlot deploymentSlotClient;
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/WebAppDeploymentSlot.java:26]: deploymentSlotInner

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/AppServicePlan.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import com.microsoft.azure.toolkits.appservice.service.IWebApp;
+
+import java.util.List;
+
+public class AppServicePlan implements IAppServicePlan {
+
+    private AppServicePlanEntity entity;
+    private AzureAppService azureAppService;
+    private AzureResourceManager azureClient;
+    private com.azure.resourcemanager.appservice.models.AppServicePlan appServicePlanClient;
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/AppServicePlan.java:23]: appServicePlanInner

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/WebAppDeploymentSlot.java (hunk 2 above)
Comment (Flanker32) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/WebAppDeploymentSlot.java:26]: Fixed in #1288

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/AppServicePlan.java (hunk 3 above)
Comment (Flanker32) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/impl/AppServicePlan.java:23]: Fixed in #1288

**Review by andxu — APPROVED** [review]

**Pull request merged**